supernode/services/userdataprocess/node.go

```diff
@@ -0,0 +1,64 @@
+package userdataprocess
+
+import (
+	"context"
+	"time"
+
+	"github.com/pastelnetwork/gonode/supernode/node"
+)
+
+const (
+	connectToNodeTimeout = time.Second * 1
+)
+
+// Nodes represents muptiple Nodes
+type Nodes []*Node
```

Comment:
Need to fix this type of structure later. Currently they are not thread safe.

Reply:
noted, thank you

Reply:
Noted.